src/test/java/org/fundacionjala/coding/Movies/TestCustomer.java

```diff
@@ -0,0 +1,188 @@
+    static final String NEW_RELEASE_MOVIE_TITLE = "Avengers";
+    static final String CHILDREN_MOVIE_TITLE = "Little Pig";
+
+    static final int DAYS_RENT_1 = 1;
+    static final int DAYS_RENT_2 = 2;
+    static final int DAYS_RENT_3 = 3;
+    static final int DAYS_RENT_5 = 5;
+    static final int DAYS_RENT_6 = 6;
+    static final int DAYS_RENT_10 = 10;
+
+    static final int RENT_AMOUNT_4 = 4;
+    static final int RENT_AMOUNT_10 = 10;
+    static final int RENT_AMOUNT_15 = 15;
+
+    static final int RENT_POINTS_2 = 2;
+    static final int RENT_POINTS_3 = 3;
+    static final int RENT_POINTS_4 = 4;
+    /**
+     * .
+     */
+    @Test
+    public void testCustomerName() {
+        Customer customerTest = new Customer("Juan");
+        assertEquals("Juan", customerTest.getName());
+    }
+
+    /**
+     * .
+     */
+    @Test
+    public void testCustomerRentalMovieBatmanFor2Days() {
+        Customer customerTest = new Customer("test");
+        Movie movie = MovieFactory.createRegularMovie(REGULAR_MOVIE_TITLE);
+        customerTest.rentMovie(movie, DAYS_RENT_2);
+        assertEquals("Batman", customerTest.getRentedMoviesList().get(0).getMovie().getTitle());
+        assertEquals(DAYS_RENT_2, customerTest.getRentedMoviesList().get(0).getDaysRented());
+    }
+
+    /**
+     * .
+     */
+    @Test
+    public void testCustomerGetRentMoviesTitles() {
+        Customer customerTest = new Customer("test");
+        Movie movie1 = MovieFactory.createRegularMovie(REGULAR_MOVIE_TITLE);
+        customerTest.rentMovie(movie1, DAYS_RENT_1);
+        Movie movie2 = MovieFactory.createRegularMovie("Superman");
+        customerTest.rentMovie(movie2, DAYS_RENT_1);
+        Movie movie3 = MovieFactory.createRegularMovie("Iron Man");
+        customerTest.rentMovie(movie3, DAYS_RENT_1);
+        assertEquals("Batman", customerTest.getRentedMoviesList().get(0).getMovie().getTitle());
+        assertEquals("Superman", customerTest.getRentedMoviesList().get(1).getMovie().getTitle());
+        assertEquals("Iron Man", customerTest.getRentedMoviesList().get(2).getMovie().getTitle());
+    }
+
+    /**
+     * .
+     */
+    @Test
+    public void testRent2DaysGetRentAmountForRegularMovie() {
+        Customer customerTest = new Customer("juan");
+        Movie regularMovie = MovieFactory.createRegularMovie("Batman");
+        customerTest.rentMovie(regularMovie, DAYS_RENT_2);
+        //rent regular: 2 per day, if(daysRented>2)--> 1.5 per day;
+        //(2*2) = 4
+        assertEquals(RENT_AMOUNT_4, customerTest.getTotalBillAmount());
+    }
+
+    /**
+     * .
+     */
+    @Test
+    public void testRent6DaysGetRentAmountForRegularMovie() {
+        Customer customerTest = new Customer("juan");
+        Movie regularMovie = MovieFactory.createRegularMovie("Batman");
+        customerTest.rentMovie(regularMovie, DAYS_RENT_6);
+        //rent regular: 2 per day, if(daysRented>2)-->  1.5 per day;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)